ons_alpha/jinja2/templates/pages/bulletins/bulletin_page.html

```diff
@@ -1,85 +1,10 @@
-{% extends "templates/base_page.html" %}
-
-{% from "components/table-of-contents/_macro.njk" import onsTableOfContents %}
+{% extends "base_page.html" %}
 
 {% block main %}
-    <div class="ons-u-fs-m ons-u-mt-s ons-u-pb-xxs release__document-type">Bulletin</div>
-    <h1 class="ons-u-fs-xxxl ons-u-mb-m">
-        <span>{{ page.title }}</span>
-        {% if page.is_accredited %}
-            <a href="https://uksa.statisticsauthority.gov.uk/about-the-authority/uk-statistical-system/types-of-official-statistics/" class="national-statistics__link ons-u-ml-s">
-                <img src="https://cdn.ons.gov.uk/assets/images/ons-logo/kitemark/v2/uksa-kitemark-en.svg" alt="UK Statistics Authority Kitemark" class="osr__logo" width="90">
-            </a>
-        {% endif %}
-    </h1>
-
-    <div class="ons-grid ons-u-ml-no">
-        {% block release_meta %}
-            <div class="ons-grid__col ons-grid__col--sticky@m ons-col-4@m ons-col-12@s ons-u-p-no">
-                <span class="ons-u-fs-r--b">{{ _("Release date") }}:</span>
-                <span class="ons-u-nowrap">{{ page.release_date|date("j F Y") }}</span>
-            </div>
-            {% if page.next_release_date %}
-                <div class="ons-grid__col ons-col-8@m ons-col-12@s ons-u-p-no@xxs@m">
-                    <span class="ons-u-fs-r--b">{{ _("Next release") }}:</span>
-                    <span class="ons-u-nowrap">{{ page.next_release_date|date("j F Y") }}</span>
-                </div>
-            {% endif %}
-
-        {% endblock release_meta %}
+    {{ table_of_contents(page, "body", {"title": "Table of contents"}) }}
+    {% include_block page.updates %}
+    {% include_block page.body %}
+    <div class="previous-releases">
+        <a href="{% pageurl page.get_parent() %}previous-releases/">Previous Releases</a>
```

Comment:
```suggestion
        <a href="{% routablepageurl page.get_parent() "previous_release" %}">Previous Releases</a>
```

as per https://docs.wagtail.org/en/stable/reference/contrib/routablepage.html#the-routablepageurl-template-tag